articles/sso/current/index.md

```diff
@@ -71,8 +71,7 @@ For more detailed information on how to implement this, please refer to the foll
 * [Client-Side SSO (Single Page Apps)](/sso/current/single-page-apps-sso)
 
 ::: note
-Please see the [Auth0 SSO Sample](https://github.com/auth0/auth0-sso-sample) repo for an example of SSO with both Single Page Apps and Regular Web Apps.
-:::
+Please see the [Auth0 OIDC SSO Sample](https://github.com/auth0-samples/oidc-sso-sample) repo for an example of how to implement OIDC-compliant SSO.
```

Comment:
Forgot some markup:

![image](https://user-images.githubusercontent.com/11715799/31656828-7f22e4fc-b335-11e7-9fb8-99de2824cb66.png)

Keep in mind to always check the review app, especially when you are editing visual elements like notes and panels.